.github/actions/execute-assert-arc-e2e/action.yaml

```diff
@@ -0,0 +1,151 @@
+name: 'Execute and Assert ARC E2E Test Action'
+description: 'Queue E2E test workflow and assert workflow run result to be succeed'
+
+inputs:
+  github-auth-token:
+    description: 'GitHub access token to queue workflow run'
+    required: true
+  github-repo-owner:
+    description: "The repository owner name that has the test workflow file, ex: actions"
+    required: true
+  github-repo-name:
+    description: "The repository name that has the test workflow file, ex: test"
+    required: true
+  github-workflow-file:
+    description: 'The file name of the workflow yaml, ex: test.yml'
+    required: true
+  test-arc-name:
```

Comment:
Is `test` really needed as part of the name? I also feel the name is not very indicative of what this is

Reply:
it's the name of the test arc we set up, any suggestions on the name? just `arc-name`?